pyproject.toml

```diff
@@ -67,6 +67,8 @@ base-runtime = [
     "Werkzeug>=3.0.0",
     "xmltodict>=0.13.0",
     "rolo>=0.4",
+    # allow Python programs full access to Java class libraries. Used for opt-in event ruler.
+    "JPype1>=1.5.0",
```

Comment:
431KB without any extra dependencies (see their [pyproject.toml](https://github.com/jpype-project/jpype/blob/master/pyproject.toml#L31)), just `packaging` and `typing_extensions`, which we have as well

Reply:
could we move it to `runtime` instead? `base-runtime` is the base one used by the S3 image, which has no dependency basically. Or was there a specific reason to put into `base-runtime`?

Reply:
That makes a lot of sense @bentsku 💯 Thank you for this valuable input 👀  👏 
I fixed everything (including the follow-up S3 tests because the test deps include the runtime deps 😬 ; that's a bit of an ugly hack though)

I didn't understand the full implications of `base-runtime` vs `runtime` dependencies.

With the fixed solution, the S3 image does not get larger 👌